staging/2019/measures-data-ia.json

```diff
@@ -156,7 +156,7 @@
   {
     "category": "ia",
     "title": "Population empanelment",
-    "description": "Empanel (assign responsibility for) the total population, linking each patient to a MIPS eligible clinician or group or care team.Empanelment is a series of processes that assign each active patient to a MIPS eligible clinician or group and/or care team, confirm assignment with patients and clinicians, and use the resultant patient panels as a foundation for individual patient and population health management. Empanelment identifies the patients and population for whom the MIPS eligible clinician or group and/or care team is responsible and is the foundation for the relationship continuity between patient and MIPS eligible clinician or group /care team that is at the heart of comprehensive primary care. Effective empanelment requires identification of the \"active population\" of the practice: those patients who identify and use your practice as a source for primary care. There are many ways to define \"active patients\" operationally, but generally, the definition of \"active patients\" includes patients who have sought care within the last 24 to 36 months, allowing inclusion of younger patients who have minimal acute or preventive health care.",
+    "description": "Empanel (assign responsibility for) the total population, linking each patient to a MIPS eligible clinician or group or care team.\rEmpanelment is a series of processes that assign each active patient to a MIPS eligible clinician or group and/or care team, confirm assignment with patients and clinicians, and use the resultant patient panels as a foundation for individual patient and population health management. \rEmpanelment identifies the patients and population for whom the MIPS eligible clinician or group and/or care team is responsible and is the foundation for the relationship continuity between patient and MIPS eligible clinician or group /care team that is at the heart of comprehensive primary care. Effective empanelment requires identification of the \"active population\" of the practice: those patients who identify and use your practice as a source for primary care. There are many ways to define \"active patients\" operationally, but generally, the definition of \"active patients\" includes patients who have sought care within the last 24 to 36 months, allowing inclusion of younger patients who have minimal acute or preventive health care.",
```

Comment:
You'll need to resolve these `\r` glitches before we can merge this into the measures data repo. It makes tracking issues in PR more difficult and could cause issues when displaying on the UI.